hello.css

```diff
@@ -0,0 +1,9 @@
+<<<<<<< Updated upstream
+.hd {
+=======
+.hdng {
+>>>>>>> Stashed changes
+    justify-content: center;
+    font-family: 'Times New Roman', 'Times New Roman', Times, serif;
+    color: blue;
+}
```

Comment:
- [ ] Please, clean your CSS file and leave only the class you want for the h1 tag like this
```
.hdng {
    justify-content: center;
    font-family: 'Times New Roman', 'Times New Roman', Times, serif;
    color: blue;
}
```